diamm/helpers/solr_pagination.py

```diff
@@ -41,27 +41,22 @@ def __init__(self, query, filters, sorts, request, *args, **kwargs):
             self.qopts.update(sorts)
 
         if filters:
-            fqstring = ""
+            fqlist = list() 
```

Comment:
Why are you converting this to a list? It would be helpful if you provided more information for your reasons in the Pull Request description.